{"name": "test_api_login_and_get_user", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'TestDataReader' has no attribute 'get'", "trace": "@allure.feature(\"API Testing\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_api_login_and_get_user():\n>       auth_api = AuthAPI()\n                   ^^^^^^^^^\n\ntests\\test_api_login.py:9: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <api.auth_api.AuthAPI object at 0x0000028E66D95160>\n\n    def __init__(self):\n        self.session = requests.Session()\n>       self.base_url = TestDataReader.get(\"base_url\")\n                        ^^^^^^^^^^^^^^^^^^\nE       AttributeError: type object 'TestDataReader' has no attribute 'get'\n\napi\\auth_api.py:7: AttributeError"}, "start": 1752589429442, "stop": 1752589429442, "uuid": "fc291786-e519-4173-8c2c-b7dab586564d", "historyId": "010deb8cff83d76073d96785733bd71d", "testCaseId": "010deb8cff83d76073d96785733bd71d", "fullName": "tests.test_api_login#test_api_login_and_get_user", "labels": [{"name": "feature", "value": "API Testing"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api_login"}, {"name": "host", "value": "HPVictus_16"}, {"name": "thread", "value": "1000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_login"}]}